### DISCUSSION TITLE
Roadmap

### DISCUSSION THREAD

**Discussion by Gazook89** [naturalcrit/homebrewery#1487] [Ideas]
There has been more gitter discussion about the user page (particularly around organization of brews) lately and so I collected issues related to the User Page below.  Perhaps it'd be a good idea to create github tags for "User Page"?   

And then I have a nagging feeling that any changes related to this topic should be reserved for a future "version" rather than trying to squeeze it into v3, which is focused on new markdown & renderer.  Correct me if I'm wrong, but I don't think HB has an official roadmap-- having one would be helpful so that A) we can organize our Issues better, B) make sure new versions are actually pushed out without feature creep, and C) increase user confidence in development.

Issues related to the User Page:
- #834 - Starred/Bookmarked brews (including others' brews).
- #697 - Published Brews (what does *published* mean?).
- #562 - Brew sorting AND unpublish vs delete button on currently published brews  (step between unpublished, published, unpublished, deleted).
- #406 - View counts reset after editing.
- #240 - aesthetic changes to user page, followed by more discussion of sorting, tagging, and custom user stylesheets for user page & folders.

Discussions:
- #1471  - Grouping

**Comment by calculuschild** [comment]
Yes, any of this can be addressed and added at any time. V3 is only for the new renderer as you say but is just being built in the background and shouldnt prevent any of this type of thing from being released.

We do have a "roadmap" milestone tracker but haven't been very good at updating it.

**Comment by G-Ambatte** [comment]
Issue #834: PR #1493 is currently the initial work on this issue; it's not ready for merging yet but a way forward has been discussed.
Issue #562: Probably should be resolved now, as #1459 User Page Ordering has been merged.
Issue #406: Should be resolved once PR #1543 is merged.

**Comment by Gazook89** [comment]
Closing this in favor of a more focused discussion ("v4").  3 out of 5 issues mentioned here are closed/completed.
